spec/requests/application_controller_spec.rb

```diff
@@ -1308,6 +1308,7 @@ def preloaded_json
             "topicTrackingStates",
             "topicTrackingStateMeta",
             "fontMap",
+            "enabledPluginAdminRoutes",
```

Comment:
This is just testing that the key exists but I think it is important to test the content as well.